packages/@tinacms/paginate/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@tinacms/paginate",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    ".env"
+  ],
+  "license": "Apache-2.0",
+  "buildConfig": {
+    "entryPoints": [
+      {
+        "name": "src/index.ts",
+        "target": "node"
+      }
+    ]
+  },
+  "devDependencies": {
+    "@tinacms/scripts": "workspace:*",
```

Comment:
I think jest needs to be a devDep (I _think_ that is why the test is failing)

Reply:
Ah, thanks!
I didn't think I needed it cause I don't have any tests, but that makes sense.